mne/viz/_3d.py

```diff
@@ -1698,6 +1701,8 @@ def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
         data = getattr(stc, hemi + '_data')
         vertices = stc.vertices[hemi_idx]
         if len(data) > 0:
+            if transparent is None:
+                transparent = True
```

Comment:
why?

Reply:
Again, if I remember well, `brain.add_data(**kwargs)` called later was failling with transparent set to None.